src/site/stages/build/drupal/graphql/page.graphql.js

```diff
@@ -1,8 +1,32 @@
+const { alert, FIELD_ALERT } = require('./block-fragments/alert.block.graphql');
+const collapsiblePanel = require('./paragraph-fragments/collapsiblePanel.paragraph.graphql');
+const {
+  listOfLinkTeasers,
+  FIELD_RELATED_LINKS,
+} = require('./paragraph-fragments/listOfLinkTeasers.paragraph.graphql');
+const process = require('./paragraph-fragments/process.paragraph.graphql');
+const qaSection = require('./paragraph-fragments/qaSection.paragraph.graphql');
+const wysiwyg = require('./paragraph-fragments/wysiwyg.paragraph.graphql');
+
 /**
  * A standard content page, that is ordinarily two-levels deep (a child page of a landingPage)
  * For example, /health-care/apply.
  */
+
+const WYSIWYG = '...wysiwyg';
```

Comment:
my thought was to have these in case the fragments change (the `...` part) we don't have to search within the big query to update it - that was just a thought, though. if we don't like it or it feels like indirection, i can just put them in as they are within the larger string 😄

Reply:
Ah, ok. I'm fine leaving them this way, we can see how it goes over time.